index.js

```diff
@@ -14,6 +13,10 @@ document.getElementById('startButton').addEventListener('click', () => {
     if (countries.length) {
         startGame()
         country = getAndDisplayCountry(countries)
+        timerObject = startTimer(timerObject)
+        setInterval(()=>{
+            console.log(timerObject.time)
```

Comment:
should this be removed, was it just for testing?

Reply:
yh it should be removed